### PR TITLE
Sovereign chain - v1

### PR DESCRIPTION
**Pull request by schimih**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

**Review by miiu96 — DISMISSED** [review]

File: docs/sovereign/concept.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- Functionality: Users can deposit tokens and specify the destination address and execution call.
+- Endpoint: `deposit@address@functionToCall@arguments@gasLimit` receiving a `MultiESDTNFTTransfer`.
+- Verification: The contract verifies the validity of the address and generates a specific `logEvent`.
+
+`logEvent` Structure
+```init
+Identifier: deposit
+Address: scAddress
+Topics: address, LIST<tokenID, nonceAsBytes, valueAsBytes/ESDTTransferData>
+Data: localNonce (increasing), functionToCall, Arguments, gasLimit
```
Comment (axenteoctavian) [inline docs/sovereign/concept.md:113]: localNonce (increasing), originalSender, gasLimit, functionToCall, arguments

File: docs/sovereign/key-components.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+This documentation is not complete. More content will be added once it is accepted and discussed on Agora or once it is implemented and available for production.
+
+Initial Sovereign Shard MIPs have been published on Agora and were used as specifications:
+- 
+
+:::
+
+Sovereign Chains mark significant progress for MultiversX in the field of appchains. While the concept is straightforward to discuss in general terms, developers, builders, or other entities looking to launch a sovereign chain need to consider several critical components. For clarity, we will divide these components into four major categories and describe the importance and role of each one.
+
+![img](/sovereign/sovereign-1.png)
```
Comment (axenteoctavian) [inline docs/sovereign/key-components.md:14]: `genesis sc` needs to be removed from cross-chain SCs

File: docs/sovereign/setup.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
+# Setup Guide
+
+:::note
+ This guide is a preliminary version and not the final documentation for sovereign chains. It serves as a starting point for setting up a sovereign chain on a local machine.
+:::
+
+This guide will help you set up and deploy contracts on a sovereign chain. Follow these steps carefully to ensure a successful deployment.
+
```
Comment (axenteoctavian) [inline docs/sovereign/setup.md:8]: Maybe add a message to look at prerequisites before starting the setup

File: docs/sovereign/setup.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+    - **"T"** for Testnet;
+- **ESDT_SAFE_WASM, FEE_MARKET_WASM, MULTISIG_VERIFIER_WASM** - represent the paths to the location where the contracts have been downloaded at Step 2.
+:::
+
+2. Source the script:
+    ```bash
+    source script.sh
+    ```
+
+3. Deploy all bridge contracts, Sovereign Notifier and Service:
```
Comment (axenteoctavian) [inline docs/sovereign/setup.md:90]: Deploy all bridge contracts, Sovereign Notifier and update Sovereign Configs

File: docs/sovereign/setup.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+    ```bash
+    deployMainChainContractsAndSetupObserver
+    ```
+
+4. Initialize and deploy all sovereign chain dependencies:
```
Comment (axenteoctavian) [inline docs/sovereign/setup.md:95]: Initialize and deploy sovereign chain with all services required

File: docs/sovereign/concept.md (hunk 1 above)
Comment (schimih) [reply docs/sovereign/concept.md:113]: done

File: docs/sovereign/setup.md (hunk 3 above)
Comment (schimih) [reply docs/sovereign/setup.md:8]: done

File: docs/sovereign/token-management.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+1. Update the `/config/configs.cfg` file with the token settings. Example:
+    ```ini
+    # Issue Token Settings
+    TOKEN_TICKER=TKN
+    TOKEN_DISPLAY_NAME=Token
+    NR_DECIMALS=18
+    INITIAL_SUPPLY=111222333
+    ```
+
+2. Navigate to the `sovereignBridge` directory:
```
Comment (axenteoctavian) [inline docs/sovereign/token-management.md:14]: Put this as step 1

File: docs/sovereign/token-management.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+1. Update the `/config/configs.cfg` file with the sovereign token settings. Example:
+    ```ini
+    # Issue Sovereign Token Settings
+    TOKEN_TICKER_SOVEREIGN=SVN
+    TOKEN_DISPLAY_NAME_SOVEREIGN=SovToken
+    NR_DECIMALS_SOVEREIGN=18
+    INITIAL_SUPPLY_SOVEREIGN=333222111
+    ```
+
+2. Navigate to the `sovereignBridge` directory:
```
Comment (axenteoctavian) [inline docs/sovereign/token-management.md:45]: Put this as step 1

**Review by axenteoctavian — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]

**Pull request merged**